### PR TITLE
Dataset: images transform

### PR DESCRIPTION
**Pull request by lucaswerner90**
Created the transform pipeline for the dataset images

### CODE REVIEW

File: dataset.py [hunk 1]
```diff
@@ -1,30 +1,77 @@
...
 
-	def __init__(self, dataset_folder='./dataset'):
+	def __init__(self, dataset_folder='./dataset',transform=None):
+		super().__init__()
+		self.transform = transform
 		self.images_folder = os.path.join(dataset_folder,'images')
-		self.captions_file = open(os.path.join(dataset_folder,'captions.txt'),'r')
-		self.dataset = [line.strip() for line in self.captions_file.readlines()[1:]]
-		self.captions_file.close()
+		self.dataframe = pd.read_csv(open(os.path.join(dataset_folder,'captions.txt'),'r'))
```
Comment (lucaswerner90) [inline dataset.py:12]: use of pandas to read the txt file

File: dataset.py [hunk 2]
```diff
@@ -1,30 +1,77 @@
...
+		transforms.Normalize(mean=[0.5, 0.5, 0.5], std=[0.5, 0.5, 0.5]),
+	])
+	dataset = Flickr8kDataset(transform=transform)
+
+	# Test the dataloader
+	dataloader = DataLoader(dataset=dataset,batch_size=batch_size)
+	batch_sample = next(iter(dataloader))
+	images, captions = batch_sample
+
+	assert images.size() == (batch_size,) + (3,) + image_size
```
Comment (lucaswerner90) [inline dataset.py:73]: I've added a couple of simple asserts to check that everything works as expected

**Comment by amolero6** [comment]
It seems great for me. I would input directly what is under if __name__ to a main.py new file

**Review by amolero6 — APPROVED** [review]

**Review by Raigm — COMMENTED** [review]
Looking good

**Pull request merged**